test/powershell/Language/Scripting/NativeExecution/NativeCommandArguments.Tests.ps1

```diff
@@ -43,4 +43,21 @@ Describe "Native Command Arguments" -tags "CI" {
             $lines[$i] | Should Be "Arg $i is <$($expected[$i])>"
         }
     }
+
+    It "Should handle PowerShell arrays with or without spaces correctly: <arguments>" -TestCases @(
+        @{arguments = "a 1,2"; expected = "a", "1,2"}
```

Comment:
let's add something boring, like `a,1,2` here as well

Reply:
There is no difference between 2 and 3 elements - it's just spaces around the argument(s) that matter.

Reply:
Should we add `1,2 3`?

Reply:
I really didn't need `a 1,2`, `1,2` would have been sufficient, but I had a bug in the test with array vs. scalar and the first thought I had was to add `a ` to force array. I'm fixing by using `@()` instead to force array.